### ISSUE TITLE
Rule proposal: disallow multiple expressions in template event listeners

### ISSUE THREAD

**Issue by MillerSvt** [angular-eslint/angular-eslint#3069]
## Description

I would like to propose a new template lint rule that disallows multiple expressions inside a single Angular template event listener.

Invalid:

```html
<button (click)="save(); close()"></button>
```

Valid:

```html
<button (click)="onClick()"></button>
```

```ts
onClick(): void {
  this.save();
  this.close();
}
```

## Motivation

Multiple expressions inside one event listener make template handlers harder to reason about and create subtle problems for async error handling.

For example:

```html
<button (click)="async1(); async2()"></button>
```

In JavaScript/Angular expression semantics, only the last expression can be observed as the listener return value. If Angular or application-level tooling wants to handle a returned `Promise`, only `async2()` is visible. The promise returned by `async1()` becomes a floating promise. (related https://github.com/angular/angular/issues/69228)

This is problematic for zoneless applications and embedded Angular applications/Web Components where relying on a global `unhandledrejection` listener is not always acceptable, because it may also capture unrelated promise rejections from the host page.

A rule that enforces a single expression per listener would encourage moving compound logic into a component method, where normal TypeScript linting rules such as `no-floating-promises` can work reliably.

## Proposed rule

Rule name idea:

```txt
template/no-multiple-event-listener-expressions
```

or:

```txt
template/no-multiple-listener-expressions
```

The rule would report Angular template listeners whose AST handler is a `Chain` with more than one expression.

Invalid:

```html
<button (click)="a(); b()"></button>
<input (change)="markDirty(); save()" />
```

Valid:

```html
<button (click)="onClick()"></button>
<input (change)="onChange($event)" />
```

## Prior art / related rules

`@angular-eslint/template/no-call-expression` is related, but it is broader and does not specifically address compound event listener expressions. This proposal is specifically about disallowing multiple expressions in a single event listener, not about banning calls in templates in general.

## Why this belongs in angular-eslint

The Angular template parser already exposes event listeners via `BoundEvent`, and compound expressions can be identified structurally through the parsed template AST. This makes the rule straightforward to implement accurately without relying on string matching.

The rule would also support a common style recommendation: keep template listeners simple and delegate multi-step logic to component methods.
